apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/AppConfiguration.java

```diff
@@ -167,6 +167,11 @@ public final class AppConfiguration extends BaseAppConfiguration {
   public static final String SSM_PATH_RIF_JOB_QUEUE_SIZE_MULTIPLE_CLAIMS =
       "ccw/job/claims/queue_size_multiple";
 
+  /**
+   * Path of the SSM parameter containing name of SQS queue to which progress messages can be sent.
+   */
+  public static final String CCW_JOB_SQS_STATUS_QUEUE_NAME = "ccw/job/sqs/status_queue_name";
+
```

Comment:
This is the new (optional) configuration setting that tells the pipeline to send status messages to an SQS queue.